layouts/layout.html

```diff
@@ -11,6 +11,12 @@
 
 	<!-- Senna is included here so it is loaded before the page loads. -->
 	<script src="https://cdn.jsdelivr.net/senna.js/2.0.4/senna-min.js"></script>
+
+	<link rel="canonical" href="https://rocket.chat/docs{{path}}" />
+
+	{{#if canonical}}
+		<link rel="canonical" href="https://rocket.chat/docs{{canonical}}" />
```

Comment:
Will this effect anything having two Canonical links?

Reply:
you mean negative?

Reply:
Yes seems like it would have negative effects having multiple.  Since by definition its pointing to the "real" version right? If so maybe the other one goes in an else?

Reply:
unless in search engine eyes second one overrides the first?

Reply:
idk.. let me google that

Reply:
https://webmasters.googleblog.com/2013/04/5-common-mistakes-with-relcanonical.html
> Specify no more than one rel=canonical for a page. When more than one is specified, all rel=canonicals will be ignored.